packages/elements/src/components/breadcrumb.ts

```diff
@@ -66,4 +69,14 @@ export class MutationTestReportBreadcrumbComponent extends LitElement {
       >
     </li>`;
   }
+
+  #renderSearchIcon() {
+    return html`
+      <button @click="${() => this.#dispatchFilePickerOpenEvent()}" class="ml-auto" aria-label="open file picker"> ${searchIcon} </button>
+    `;
+  }
+
+  #dispatchFilePickerOpenEvent() {
+    this.dispatchEvent(new CustomEvent('mte-file-picker-open'));
```

Comment:
Very minor thing, but if you call `createCustomEvent` instead of new CustomEvent you'll get type-safe events and args (which are void here, but still)